lib/Varien/Io/Sftp.php

```diff
@@ -117,7 +115,7 @@ public function rmdir($dir, $recursive = false)
             $list = $this->_connection->nlist();
             if (!count($list)) {
                 // Go back
-                $this->_connection->chdir($pwd);
+                $this->_connection->chdir($cwd);
```

Comment:
Is it better to change line 111 instead:

from `$cwd = $this->_connection->pwd();`
to `$pwd = $this->_connection->pwd();`

Reply:
I think `pwd` means "print working directory", and `cwd` means "current working directory". So `cwd` is the value resulting from calling `pwd()`.